routes/login.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const User = require('../models/User');
+const router = express.Router();
+
+// create a response for the homepage
+router.post('/', (req, res , next) => {
+    User.create(req.body, (err ,user) => {
+        if(err){
+            res.json(
+              {
+                configuration: 'fail',
+                error : err,
+            }  
```

Comment:
Nitpick: formatting - would be great to have the intention of the brackets aligned to have a consistent code styling.